### PR TITLE
Add duration in config

### PR DESCRIPTION
**Pull request by FanchTheSystem**

### CODE REVIEW

File: manager2/src/app/auth/register/register.component.html [hunk 1]
```diff
@@ -70,11 +70,7 @@
             <div class="form-group">
                 <label for="duration">Duration</label>
                 <select id="duration" [ngModelOptions]="{standalone: true}" [(ngModel)]="duration" class="form-control" required>
-                    <option value="90">3 months</option>
-                    <option value="180">6 months</option>
-                    <option value="365">1 year</option>
-                    <option value="730">2 years</option>
-                    <option value="1095">3 years</option>
+                  <option *ngFor="let duration of config.duration">{{duration}}</option>
```
Comment (osallou) [inline manager2/src/app/auth/register/register.component.html]: should set value of option, will display in "month/year" but api expects days

Comment (FanchTheSystem) [reply manager2/src/app/auth/register/register.component.html]: I don't think is a good idéa to have the day in frontend (anyone could set it to 999999999999 ....) 

So I have modified duration to be a string, so it contain only user input, and expiration is calculated using this input to find the number of day

Comment (FanchTheSystem) [reply manager2/src/app/auth/register/register.component.html]: Do you need a script to update duration in mongo ?

Comment (osallou) [reply manager2/src/app/auth/register/register.component.html]: I don't know if it is necessary, duration is only at registration time only I think.
Or is it reused when extending/reactivating user (admin or self-extend) ? Which duration is used at this time ?

Comment (osallou) [reply manager2/src/app/auth/register/register.component.html]: Arrg, extend is using user duration field....

            let expiration = new Date().getTime() + 1000*3600*24*user.duration;

so it needs to be updated in db..... 
The problem to update in db is if there are some custom durations....

Comment (FanchTheSystem) [reply manager2/src/app/auth/register/register.component.html]: I will add a script which find the nearest duration if it is custom value.
(I will do it tomorow)

Comment (FanchTheSystem) [reply manager2/src/app/auth/register/register.component.html]: here is the script: https://github.com/genouest/genouestaccountmanager/pull/148/commits/cab37ade5909d2e865a055ce5facfe9bb558d193

I have tested it with some random generate number

**Comment by FanchTheSystem** [comment]
So I think this is PR is ok for me, please merge or comment ;)

**Comment by osallou** [comment]
you have a conflict on conf.js

**Comment by FanchTheSystem** [comment]
conflict resolved, please could you tag after this PR ?

**Comment by osallou** [comment]
tagged 1.3.0

**Comment by FanchTheSystem** [comment]
> tagged 1.3.0

Thank you ;)

**Pull request merged**